### ISSUE TITLE
Games

### ISSUE THREAD

**Issue by badass-boss** [74644323/OPN3LPT#1]
Will this work with PC-98 games?

**Comment by 74644323** [comment]
Yes, it works on pc-9821 laptop,with PC98 DOS games that using P.M.D as music driver,such as Touhou 2,3,4,5
Programs and games that use other music drivers like MDRV2 or FMP are currently not supported.

**Comment by 74644323** [comment]
If I have enough spare time in the next year or two, I may continue to improve the PMDLPT driver, or write a DOS universal IO port trap TSR driver based on EMM386 protected mode like adlipt. If this idea succeeds, it will be able to support most PC98 DOS games and software.

**Comment by badass-boss** [comment]
Yes I support the idea of having a universal TSR driver for this just like OPL2LPT! This device would have a gigantic potential and would turn PC-9821 laptops with no audio to being perfectly usable machines for PC-98 gaming

**Comment by badass-boss** [comment]
Kind of related and unrelated question- does OPN3LPT work with Totsugeki Mix? I just can't for the life of me get this to work on PC-9821NR233... As far as I can tell this game uses some sort of custom PMD driver but the compatibility is just horrible. So, does OPN3LPT support it or do you have any ideas how to make this work on NR233 or any other machine using this sound system?
